resources/js/electron-plugin/src/server/api/childProcess.ts

```diff
@@ -123,10 +137,10 @@ router.post('/stop', (req, res) => {
     res.sendStatus(200);
 });
 
-router.post('/restart', (req, res) => {
+router.post('/restart', async (req, res) => {
     const {alias} = req.body;
 
-    const settings = getSettings(alias);
+    const settings = {...getSettings(alias)};
```

Comment:
I don't understand the value of this - wrapping in an object and then unpacking the object that comes back from `getSettings` seems redundant

Reply:
Ah! That surprises me. Ok, in that case this seems fine